testfiles/HybridModel_4Zone.idf

```diff
@@ -0,0 +1,1459 @@
+! 4ZoneHybridModel.idf
```

Comment:
Change to match the name of the file.  Also, this file is full of tabs.
Looking at the other test files, I would name this "HybridZoneModel" to distinguish it from the HybridVentilation... test files.  Also, you need to add a line for this to testfiles/CMakeLists.txt